docs/dev-guides/index.md

```diff
@@ -0,0 +1,27 @@
+# GitOps Toolkit
```

Comment:
I think this should be the index of "Toolkit Components" and linked in the menu as "Overview".

Reply:
Yarp OK